husky_components/matrix/main.js

```diff
@@ -11,7 +11,8 @@ define(function() {
 
     'use strict';
 
-    var activeClass = 'is-active';
+    var activeClass = 'is-active',
+        currentSelectButton = 'td:last-child > span:last-child';
```

Comment:
That reads like a selector, not a button, so the naming is not really appropriate. And why `current`? Isn't that selector valid for all the buttons?